### ISSUE TITLE
Add checksum file(s)

### ISSUE THREAD

**Issue by sid-the-sloth** [JetpackDuba/Gitnuro#200]
Every software release is safer when checksum verification file(s) are generated during build.

Please add checksum verification files to the release (SHA256, SHA512, etc.)

I would like to be able to verify the integrity of downloaded files/binaries.

Example projects on github which are doing that:

https://github.com/yt-dlp/yt-dlp

https://github.com/keepassxreboot/keepassxc

https://github.com/VSCodium/vscodium

Thank you! This is greatly appreciated.

**Comment by JetpackDuba** [comment]
Hey!

I already do it for the flatpak and homebrew releases so it's not an issue to add it to the rest. I'll do it in a moment.

**Comment by JetpackDuba** [comment]
Done!

**Comment by sid-the-sloth** [comment]
> Done!

I saw the SHA256SUM file published yesterday for release 1.3.1.

I think there is a problem with current file:

```
ef29def5d0ffed54fe1f8b48049be939ff24b04082a683406bc4d29859a360cc linux x86_64
e934507c885a2818cd3e7ded1155a8505f762409b094752d41913c80a119fd92 linux arm aarch_64
fef126513c3b1a1f1ce29673f596547b16e31103cbc31007297f77b92e91b0eb windows installable
9332eb4183e61022981b2779d7e2ce82187a9a933a72683bb69729305ee1e28b windows portable
cb62e5921c058844d8b0a2900ae3316f86d8baacbb00d1b7c89d5e4dc40185d1 macos zip
```

The right-hand part of every line should contain the **actual filename of the binary** (e.g.: 
`Gitnuro-linux-x86_64-1.3.1.jar`) not those explanation words like `linux x86_64`:

```
ef29def5d0ffed54fe1f8b48049be939ff24b04082a683406bc4d29859a360cc  Gitnuro-linux-x86_64-1.3.1.jar 
e934507c885a2818cd3e7ded1155a8505f762409b094752d41913c80a119fd92  Gitnuro-linux-arm_aarch64-1.3.1.jar 
fef126513c3b1a1f1ce29673f596547b16e31103cbc31007297f77b92e91b0eb  Gitnuro_Windows_Installer_1.3.1.exe 
9332eb4183e61022981b2779d7e2ce82187a9a933a72683bb69729305ee1e28b  Gitnuro_Windows_Portable_1.3.1.zip 
cb62e5921c058844d8b0a2900ae3316f86d8baacbb00d1b7c89d5e4dc40185d1  Gitnuro-macos-1.3.1.zip 
```

**Comment by JetpackDuba** [comment]
Fixed!
